Makefile

```diff
@@ -104,7 +104,7 @@ checkdocstrings: python
 
 .PHONY: pip-compile
 pip-compile: python
-	tox -q -e py36-dev -- pip-compile --output-file requirements.txt requirements.in
+	tox -q -e py36-dev -- pip-compile
```

Comment:
Its a bit confusing that pip-compile updates requirements.txt as well as printing it to stdout. Usually tools do one or the other in any invocation.